### PR TITLE
Add support for reading debug information from the error details

### PR DESCRIPTION
**Pull request by josephschorr**
Reference: https://github.com/authzed/spicedb/pull/1930

### CODE REVIEW

File: internal/commands/permission.go [hunk 1]
```diff
@@ -222,8 +223,16 @@ func checkCmdFunc(cmd *cobra.Command, args []string) error {
...
+
+		// Check for the debug trace contained in the error details.
+		if errInfo, ok := grpcErrorInfoFrom(err); ok {
+			if encodedDebugInfo, ok := errInfo.Metadata["debug_trace_proto_text"]; ok {
+				debugInfo = &v1.DebugInformation{}
+				err = prototext.Unmarshal([]byte(encodedDebugInfo), debugInfo)
+				if err != nil {
+					return err
+				}
+			}
```
Comment (vroldanbet) [inline internal/commands/permission.go:235]: Let's please add a test

**Review by vroldanbet — COMMENTED** [review]
Changes look good to me, but to make sure it does not regress again, let's add a test.

File: internal/commands/permission.go (hunk 1 above)
Comment (josephschorr) [reply internal/commands/permission.go:235]: The main regression test is on the SpiceDB side but I'll see if I can add a test for parsing the proto here

Comment (josephschorr) [reply internal/commands/permission.go]: Added

**Review by vroldanbet — APPROVED** [review]

**Pull request merged**